drivers/scheduler/scheduler.go

```diff
@@ -76,6 +76,9 @@ type Driver interface {
 
 	// Describe generates a bundle that can be used by support - logs, cores, states, etc
 	Describe(*Context) (string, error)
+
+	//Scale the deployment "up" or "down" based on scaleType . Returns error if scaling failed.
```

Comment:
This should be called ScaleApp (name of the method) as a practice. Lint should have failed here.
nit: space after // 
and no space before the full stop